src/main.zig

```diff
@@ -3449,7 +3449,7 @@ fn createModule(
     index: usize,
     parent: ?*Package.Module,
     zig_lib_directory: Cache.Directory,
-) Allocator.Error!*Package.Module {
+) !*Package.Module {
```

Comment:
nope. that error set is there for a reason.

Reply:
Oh my bad, sorry for that, i re-put back this line as well ! TT